pandas/io/excel.py

```diff
@@ -777,17 +776,20 @@ def _pop_header_name(row, index_col):
 
 
 def _conv_value(val):
-    # Convert numpy types to Python types for the Excel writers.
+    """ Convert numpy types to Python types for the Excel writers.
+        :obj:`datetime` and :obj:`date` formatting must be handled in the
+        writer. :obj:`str` representation is returned for all other types.
+    """
```

Comment:
Write an expanded docstring with Parameters and Returns section if you're going to do this.